Samples/Islands/cs-wpf-packaged/WpfWithIslandApp/WpfWithIslandApp/WpfWithIslandApp.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+
+    <!--
+      Changed_from_original_app
+      The lowest version of Windows that WinAppSDK supports is 17763.
+      But Lottie requires Windows 10.0.19041.0 or higher, so we target this one.
+    -->
+    <TargetFramework>net8.0-windows10.0.19041.0</TargetFramework>
+
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <UseWPF>true</UseWPF>
+
+    <!--
+      Changed_from_original_app
+      Set RuntimeIdentifiers explicitly to portable RIDs.
+      Set the Platforms property as well.  We don't support AnyCPU because WinAppSDK contains native code.
+    -->
+    <RuntimeIdentifiers>win-x64;win-x86;win-arm64</RuntimeIdentifiers>
+    <Platforms>x64;x86;ARM64</Platforms>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- 
+      Changed_from_original_app: These packages are required for Lottie.
+      FIX BEFORE COMPLETING PR: These packages aren't currently publicly available. 
+    -->
+    <PackageReference Include="CommunityToolkit.WinAppSDK.LottieIsland" Version="0.1.1-prerelease.2" />
```

Comment:
@JesseCol, can you check with Geoffrey whether this is the correct Lottie NuGet package that he's assembled?